opendbc/safety/safety/safety_gm.h

```diff
@@ -135,7 +135,7 @@ static bool gm_tx_hook(const CANPacket_t *to_send) {
   // GAS/REGEN: safety check
   if (addr == 0x2CB) {
     bool apply = GET_BIT(to_send, 0U);
-    int gas_regen = ((GET_BYTE(to_send, 2) & 0x7FU) << 5) + ((GET_BYTE(to_send, 3) & 0xF8U) >> 3);
+    int gas_regen = (int)((((GET_BYTE(to_send, 1) & 0x7U) << 16) + (GET_BYTE(to_send, 2) << 8) + GET_BYTE(to_send, 3)) >> 3) - 22534;
```

Comment:
This would block messages since openpilot is sending a float, and this truncates it...